File: App/StudentsApp/src/Model/Student.ts
export class Student{
    public Name: string;
    public Age: number;
    public Hobbies: string[];

    constructor(name: string, age: number, hobbies: string[]){
        this.Name = name;
        this.Age = age;
        this.Hobbies = hobbies;
    }
}

File: API/Students.API/Model/Student.cs
namespace Students.API.Model
{
    /// <summary>
    /// Class that encapsulates "Student" object.
    /// </summary>
    public class Student
    {
        /// <summary>
        /// Gets or sets the student's name.
        /// </summary>
        public string? Name { get; set; }

        /// <summary>
        /// Gets or sets the student's age.
        /// </summary>
        public int Age { get; set; }

        /// <summary>
        /// Gets or sets the student's hobbies.
        /// </summary>
        public IEnumerable<string>? Hobbies { get; set; }
    }
}
